FreeSurfer/FreeSurferPipelineNHP.sh

```diff
@@ -0,0 +1,1140 @@
+
+#TSC: repeatable options aren't currently supported in newopts, do them manually and fake the help info for now
+opts_AddOptional '--extra-reconall-arg' 'extra_reconall_args' 'token' "(repeatable) Generic single token argument to pass to recon-all.  Provides a mechanism to customize the recon-all command and/or specify the recon-all stage(s) to be run (e.g., in the case of FreeSurfer edits).  If you want to avoid running all the stages inherent to the '-all' flag in recon-all, you also need to include the --existing-subject flag.  The token itself may include dashes and equal signs (although Freesurfer doesn't currently use equal signs in its argument specification).  e.g., --extra-reconall-arg=-3T is the correct syntax for adding the stand-alone '-3T' flag to recon-all, but --extra-reconall-arg='-norm3diters 3' is NOT acceptable.  For recon-all flags that themselves require an argument, you can handle that by specifying  --extra-reconall-arg multiple times (in the proper sequential fashion), e.g. --extra-reconall-arg=-norm3diters --extra-reconall-arg=3 will be translated to '-norm3diters 3' when passed to recon-all."
+
+opts_AddOptional '--conf2hires' 'conf2hires' 'TRUE/FALSE' "Indicates that the script should include -conf2hires as an argument to recon-all.  By default, -conf2hires is included, so that recon-all will place the surfaces on the hires T1 (and T2).  Setting this to false is an advanced option, intended for situations where: (i) the original T1w and T2w images are NOT 'hires' (i.e., they are 1 mm isotropic or worse), or  (ii) you want to be able to run some flag in recon-all, without also regenerating the surfaces, e.g. --existing-subject --extra-reconall-arg=-show-edits --conf2hires=FALSE" "TRUE"
+
+opts_AddOptional '--processing-mode' 'ProcessingMode' 'HCPStyleData or LegacyStyleData' "Controls whether the HCP acquisition and processing guidelines should be treated as requirements.  'HCPStyleData' (the default) follows the processing steps described in Glasser et al. (2013) and requires 'HCP-Style' data acquistion.  'LegacyStyleData' allows additional processing functionality and use of some acquisitions that do not conform to 'HCP-Style' expectations.  In this script, it allows not having a high-resolution T2w image." "HCPStyleData"
+
+# NHP options
+opts_AddMandatory '--species' 'Species' 'Human|Chimp|MacaqueCyno|MacaqueRhes|MacaqueFusc|NightMonkey|Marmoset' "Species type (required).  Controls species-specific processing parameters" ""
```

Comment:
Mandatory parameters don't take a default (optional parameters default to empty string, so you wouldn't need to specify `""` anyway, unless you were adding a legacy flag).
```suggestion
opts_AddMandatory '--species' 'Species' 'Human|Chimp|MacaqueCyno|MacaqueRhes|MacaqueFusc|NightMonkey|Marmoset' "Species type (required).  Controls species-specific processing parameters"
```